css/main.css

```diff
@@ -416,6 +441,39 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+/* mobile menu */
+.menu-visible {
+  position: fixed;
+  display: block;
+  background-color: #e5e5e5;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  padding: 0;
+}
+
+.menu-visible nav {
+  padding: 100px 1px;
+  background-color: #3c3a39;
+  height: 100vh;
+}
+
+.menu-visible li a {
+  color: white;
+}
+
+.close-btn {
+  display: block;
+  position: absolute;
+  top: 40px;
+  right: 24px;
+  font-size: 18px;
+  width: auto;
+  background-color: transparent;
+  border-width: inherit;
+  color: #f4f5f7;
+}
```

Comment:
- [OPTIONAL] Kindly, make sure your stick to the provided Figma design while you style your mobile menu. 
 
   - Your design
     
 ![image](https://user-images.githubusercontent.com/87197412/184103853-3fd402ec-9011-47a6-a666-f2896650b4f2.png)

  - Figma design
  
 ![image](https://user-images.githubusercontent.com/87197412/184105030-2b631cfb-7bbb-439c-bfe8-caf9b973dcd2.png)